src/withESI.tsx

```diff
@@ -46,7 +42,7 @@ export default function withESI<P>(
       this.esi = esi || {};
       this.state.childProps = childProps;
 
-      if (!(process as IWebpackProcess).browser) {
+      if (typeof window.navigator === 'undefined') {
```

Comment:
shouldn't it be `typeof navigator !== 'undefined'` like below?

Reply:
no, notice the `!`